crates/wysiwyg/src/dom/to_html.rs

```diff
@@ -25,11 +25,18 @@ where
         buf: &mut S,
         selection_writer: Option<&mut SelectionWriter>,
         state: ToHtmlState,
+        clean: bool,
```

Comment:
I couldn't settle on the naming of this parameter. 'Cleaning' the HTML seemed closest to what we'd be doing. I also considered 'as_internal_representation` as the inverse but it's a bit long.

Reply:
Updated to `as_message` (related to https://github.com/matrix-org/matrix-rich-text-editor/pull/686#discussion_r1203765574)